docs/pages/contributing/documentation/terms-usage.mdx

```diff
@@ -0,0 +1,171 @@
+## Terminology and usage conventions
+
+This section covers common terminology, style, and usage questions.
+It includes recommended practices for using punctuation, referring to user interface
+elements, and selecting the right words for the information you're trying to convey. 
+
+### Above and below
+
+Don't use *above* to mean *earlier* or as an adjective preceding a noun (*the above section*) 
+or following a noun (*the code above*). Use a hyperlink, or use *previous*, *preceding*, or 
+*earlier*. Don't use *below* to mean *later* or as an adjective preceding a noun (*the below 
+section*) or following a noun (*the code below*). Use a hyperlink, or use *later* or 
+*the following*. For example: 
+
+- Use the preceding code to display information about the database.
+- Use the following code to display information about the database.
```

Comment:
Would you mind elaborating on the reasoning behind this one?

Reply:
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/a/above
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/b/below